pkg/client/config.go

```diff
@@ -69,13 +69,11 @@ func (c *ClientConfig) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 }
 
 type Config struct {
-	Enabled                   bool   `yaml:"enabled" json:"enabled,omitempty" koanf:"enabled"`
-	ApplicationID             string `yaml:"application-id" json:"application-id,omitempty" koanf:"application-id"`
-	TxServiceGroup            string `yaml:"tx-service-group" json:"tx-service-group,omitempty" koanf:"tx-service-group"`
-	AccessKey                 string `yaml:"access-key" json:"access-key,omitempty" koanf:"access-key"`
-	SecretKey                 string `yaml:"secret-key" json:"secret-key,omitempty" koanf:"secret-key"`
-	EnableAutoDataSourceProxy bool   `yaml:"enable-auto-data-source-proxy" json:"enable-auto-data-source-proxy,omitempty" koanf:"enable-auto-data-source-proxy"`
```

Comment:
Are `EnableAutoDataSourceProxy` and `DataSourceProxyMode` not used in any other places?

Reply:
nope